lib/webpacker/compiler.rb

```diff
@@ -4,11 +4,27 @@ module Webpacker::Compiler
   extend self
 
   def compile
+    return unless source_changed?
+    cache_source_checksum
     compile_task.invoke
     compile_task.reenable
   end
 
   private
+    def source_checksum
```

Comment:
Not sure if checksum is the best term since it's not computing a hash. Maybe something more explicit?